src/MagicOnion.Server.OpenTelemetry/OpenTelemetryCollectorTracerFilterAttributes.cs

```diff
@@ -38,7 +38,6 @@ public override async ValueTask Invoke(ServiceContext context, Func<ServiceConte
         {
             // https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/rpc.md#grpc
 
-            // Client -> Server incoming filter
             // span name must be `$package.$service/$method` but MagicOnion has no $package.
             using var activity = source.StartActivity($"{context.MethodType}:{context.CallContext.Method}", ActivityKind.Server);
```

Comment:
Activity from `ActivitySource.StartActivity` may be null if no one listen or all listener returns ActivitySamplingResult.None.
so it should be considered `activity` may be null.

Reply:
thank you, added null check.

Reply:
with this fix, Invoke seems to exit without invoke next when Activity is null

Reply:
oops, fixed.